lit/SwiftREPL/ResilientArray.test

```diff
@@ -0,0 +1,10 @@
+// REQUIRES: system-darwin
+// RUN: %lldb --repl < %s | FileCheck %s
+
+import Foundation
+
+let x : [URL] = [URL(string: "https://github.com")!, URL(string: "https://apple.com")!]
```

Comment:
I think we've had this discussion before, but this is not the best way to test this change. It would be better to have a self-contained test that doesn't depend on Foundation and can run on all platforms, by also including the definition of the resilient type in the test.

Reply:
Yes. Fair enough. I'll add a new test. Admittedly I should've done this here. Apologies.